app/views/govuk_publishing_components/components/_step_by_step_nav_related.html.erb

```diff
@@ -1,14 +1,17 @@
 <%
   add_gem_component_stylesheet("step-by-step-nav-related")
-
+  ga4_tracking ||= false
```

Comment:
Sorry, should have thought of this sooner - this needs a test, to check that the right attributes appear when the component is passed this option. This is not only to confirm that it works, but also to raise a flag if a developer were to accidentally remove part of this code in the future.

Reply:
(not necessary to add one for the other component - since it's a wrapper for components it'd be hard to anyway)

Reply:
Cool, have added tests now. Thanks for spotting that 👍